src/task_12_2_classes.py

```diff
@@ -0,0 +1,123 @@
+"""
+Homework - 12
+task_12_2_classes: Создать класс Point, описывающий точку(атрибуты: x, y).
+Создать класс Figure. Создать три дочерних класса Circle(атрибуты: координаты
+центра(тип Point), длина радиуса; методы: нахождение периметра и площади окружности),
+Triangle(атрибуты: три точки, методы: нахождение площади и периметра),
+Square(атрибуты: две точки, методы: нахождение площади и периметра).
+При потребности создавать все необходимые методы не описанные в задании.
+Создать список фигур и в цикле подсчитать и вывести площади всех фигур на экран
+[my-oop-03]
+Примечание: в рамках задание создать два файла: classes.py и main.py .
+В первом будут описаны все классы, во втором классы будут
+импортированы и использованы.
+"""
+from math import pi
+
+
+class Point:
+    """Класс точки, описывающий в атрибутах ее координаты x и y"""
+    def __init__(self, x, y) -> None:
+        self.x = x
+        self.y = y
+
+
+class Figure:
+    """Класс фигуры с пустыми атрибутами и методами площади и периметра"""
+    def __init__(self, area_a=None, perimeter_a=None) -> None:
+        self.area_a = area_a
+        self.perimeter_a = perimeter_a
+
+    def area_m(self) -> None:
+        pass
+
+    def perimeter_m(self) -> None:
+        pass
+
+
+class Circle(Figure):
+    """Класс круг с доп. атрибутами и переопределенными методами площади и периметра"""
+    def __init__(self, a: Point, r):
+        super().__init__(Figure)
```

Comment:
https://pythononline.ru/osnovy/metod-super-python посмотри примеры использование super()

в метод ты передаешь значения а не класс. В текущем варианте твое решение работает неверно.
```suggestion
        super().__init__()
```